token-metadata/program/src/utils.rs

```diff
@@ -575,8 +584,19 @@ pub fn mint_limited_edition<'a>(
     if mint_supply != 1 {
         return Err(MetadataError::EditionsMustHaveExactlyOneToken.into());
     }
-
+    let master_data = master_metadata.data;
+    // bundle data into v2
+    let data_v2 = DataV2 {
+        name: master_data.name,
+        symbol: master_data.symbol,
+        uri: master_data.uri,
+        seller_fee_basis_points: master_data.seller_fee_basis_points,
+        creators: master_data.creators,
+        collection: master_metadata.collection,
+        uses: master_metadata.uses,
```

Comment:
You need to make a new uses where the starting uses == the max available so the limited edition gets a fresh start